gst/nnstreamer/include/tensor_typedef.h

```diff
@@ -86,6 +90,14 @@
 #define GST_TENSORS_CAP_DEFAULT \
     GST_TENSORS_CAP_WITH_NUM(GST_TENSOR_NUM_TENSORS_RANGE)
 
+/**
+ * @brief Caps string for the caps template of flexible tensors.
+ * This mimetype handles non-static, flexible tensor stream without specifying the data type and shape of the tensor.
+ * The maximum number of tensors in a buffer is 16 (NNS_TENSOR_SIZE_LIMIT).
+ */
+#define GST_FLEX_TENSORS_CAP_DEFAULT \
```

Comment:
How about a `GST_TENSORS_FLEX_CAP_DEFAULT` for consistency?

Reply:
updated